components/cli/pkg/commands/init.go

```diff
@@ -125,23 +133,18 @@ func RunInit(projectName string) {
 	if err != nil {
 		util.ExitWithErrorMessage("Error in creating Ballerina File", err)
 	}
-	defer func() {
-		err = balFile.Close()
-		if err != nil {
-			util.ExitWithErrorMessage("Failed to clean up", err)
-		}
-	}()
+	return balFile, projectDir
+}
+
+// WriteToBalFile writes the content to a bal file.
+func WriteToBalFile(balFile io.Writer, cellTemplate string) {
```

Comment:
This does not actually write to a file.. but to any type of io.Writer.. please rename to Write()?

Reply:
Updated to "writerCellTemplate"